src/routes/homepage-redesign-preview/index.svelte

```diff
@@ -1,106 +1,260 @@
+  const icons = [
+    "sun",
+    "rainfall",
+    "wildfire",
+    "snowflake",
+    "sea",
+    "streamflow",
+  ].map((name) => `img/icons/${name}.svg`);
+  const cardHeight = 18;
+  const cardWidth = 18;
 </script>
 
+<style lang="scss">
+  @import "scss/site/mixins/media-queries";
+
+  .alert :global(.bx--inline-notification) {
+    margin: 0;
+    max-width: 100%;
+    background: $teal-10;
+
+    &::before {
+      border: none;
+    }
+  }
+
+  .alert :global(.bx--inline-notification__text-wrapper) {
+    flex-direction: column;
+  }
+
+  .bx--inline-notification__subtitle {
+    margin-bottom: 0;
+  }
+
+  .btn-container {
+    margin-top: 2.5rem;
+    text-transform: uppercase;
+  }
+
+  .ae-logo-box {
+    padding: 14px;
+
+    img {
+      display: block;
+      margin: auto;
+    }
+  }
+
+  .ae-logo-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    align-items: flex-start;
+
+    @include media("<=medium") {
+      padding-top: 24px;
+    }
+  }
+
+  .cta-card {
+    width: 350px;
+
+    @include media(">large") {
+      float: right;
+    }
+
+    @include media("<=medium") {
+      margin-top: 24px;
+    }
+  }
+
+  .icon-circle {
+    svg {
+      fill: red;
+      color: red;
+    }
+  }
+</style>
+
 <svelte:head>
   <title>Cal-Adapt</title>
 </svelte:head>
 
+<!-- Banner -->
 <Banner
-  titleText="Cal-Adapt provides public access to climate data that contribute to
-    California’s Climate Change Assessments."
-  subtitleText="Cal-Adapt provides interactive, easy-to-use visualization and
-    data download tools for a broad range of stakeholders; a “pro” version for
-    users with highly technical, data-intensive needs; and links to state-funded
-    grants that support development of data for Cal-Adapt."
+  titleText="Explore and analyze climate data from California’s Climate Change Assessments"
+  subtitleText="Cal-Adapt provides the public, researchers, government agencies and industry stakeholders with essential data & tools for climate adaptation planning, building resiliency, and fostering community engagement."
+  bannerImg="/img/banners/yosemite_1600x540.jpg"
+  bannerImgMobile="/img/banners/yosemite_700x800.jpg"
+  overlayOpacity="{0.4}"
   titleFontSize="2rem"
   titleFontWeight="600"
+  iconPaths="{icons}"
   useOffset="{false}"
-  bannerImg="linear-gradient(180deg, #003850 0%, #067C74 100%)"
-  overlayOpacity="0"
-/>
+>
+  <!--   <div class="btn-container" slot="button">
+    <Button icon="{ArrowRight16}" href="/about" sapper:prefetch
+      >More about Cal-Adapt</Button
+    >
+  </div> -->
+  <div class="cta-card" slot="cta-card">
+    <Card
+      {...{
+        titleText: "Cal-Adapt is evolving!",
+        linkPath: `/help/`,
+        description:
+          "Learn about the Cal-Adapt enterprise and our mission to support California's climate change initiatives and preview our future plans.",
+        height: 18,
+        ctaText: "Read More",
```

Comment:
You can add  `bgColor: "rgba(255,255,255,0.5)"` prop to get a card with transparent background (the color I'm using here is just an example)

Reply:
Oh ok. I tried that but I think I might have forgot the ""